content/metadata-rework.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "Metadata rework"
+date = 2024-07-25
+weight = 39
+template = "page.html"
+render = true
++++
+
+*John Baublitz, Stratis Team*
+
+Until now, Stratis has had a metadata format that, with a few exceptions, works
+remarkly well for our current feature set. However, as we plan for larger features
+like RAID, integrity, and online reencryption, we've noticed that we require some
+reworks to our stacking and the addition of reservation ahead of time for new devicemapper
```

Comment:
^reworks to our stacking^adjustments to the Stratis logical device stack

Reply:
... and the addition of reservation ahead of time for new devicemapper layers.

I think starting a whole new sentence here is what you need to make it clearer.

Reply:
Resolved.